### PR TITLE
Mobile optimization

### PR DESCRIPTION
**Pull request by anehx**
* Prevent mobile users from accessing `/reschedule` and `/analysis`
* Some responsive grid changes

### CODE REVIEW

File: app/mixins/desktop-only-route.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+   * @method beforeModel
+   * @param {Ember.Transition} transition The transition
+   * @return {*} The return value of the parent function or a new transition
+   * @public
+   */
+  beforeModel(transition) {
+    if (
+      Object.values(
+        this.getProperties('media.isMd', 'media.isLg', 'media.isXl')
+      ).some(i => i)
```
Comment (czosel) [inline app/mixins/desktop-only-route.js]: This looks a bit funny, but i can't think of a good alternative... What came to mind was `.some(Boolean)` or `.some(i => !!i)` but that's also not any easier on the eye.

Comment (czosel) [reply app/mixins/desktop-only-route.js]: Well i guess most readable would be the boring way: `this.get('media.isMd') || this.get('media. ... `

File: app/reschedule/route.js [hunk 2]
```diff
@@ -65,25 +70,26 @@ export default Route.extend(StaffRouteMixin, ReportFilterRouteMixin, {
...
+          /* istanbul ignore next */
+          this.get('notify').error('Error while verifying the page')
+        } finally {
+          this.send('finished')
+        }
       }
     }
   }
-})
+)
```
Comment (czosel) [inline app/reschedule/route.js]: Prettier updates .... ^^

Comment (czosel) [reply app/reschedule/route.js]: Ah, not even. I thought this was because of the new way prettier is laying out chained function calls. nvmd

Comment (anehx) [reply app/reschedule/route.js]: Wait a second.. That's weird..

File: app/mixins/desktop-only-route.js (hunk 1 above)
Comment (anehx) [reply app/mixins/desktop-only-route.js]: Yeah you're right..

File: app/reschedule/route.js (hunk 2 above)
Comment (anehx) [reply app/reschedule/route.js]: OHGODNO that's ugly..

Comment (anehx) [reply app/reschedule/route.js]:
```javascript
const BaseRoute = Route.extend(
  StaffRouteMixin,
  ReportFilterRouteMixin,
  DesktopOnlyRouteMixin,
  {}
)

/**
 * Route for filtering and rescheduling reports
 *
 * @class RescheduleRoute
 * @extends Ember.Route
 * @using StaffRouteMixin
 * @using ReportFilterRouteMixin
 * @using DesktopOnlyRouteMixin
 * @public
 */
export default BaseRoute.extend({ ... })
```
Worse or better?

Comment (czosel) [reply app/reschedule/route.js]: We're using prettier - there is no more "ugly" or "beautiful" just "prettier-conformant" or not :stuck_out_tongue_closed_eyes:

Comment (czosel) [reply app/reschedule/route.js]: better!

**Pull request merged**